src/redux/rockets/rocketsSlice.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-param-reassign */
```

Comment:
- [ ] [REQUIRED] Kindly enable eslint. Disabling eslint will not allow it to check for the Javascript code standards. Please, ensure to fix the linter errors manually.